libs/pages/services/src/lib/ui/page-settings-deployment-pipeline/page-settings-deployment-pipeline.tsx

```diff
@@ -105,100 +156,69 @@ export function PageSettingsDeploymentPipeline(props: PageSettingsDeploymentPipe
           </Button>
         </SettingsHeading>
         <div className="h-full overflow-x-auto">
-          {!stages && !services ? (
+          {!boardData ? (
```

Comment:
Can we waiting Cards data before display the board ?

It's to avoid this case when we reload the page
https://www.loom.com/share/fa8d2ba0c99643ec9d8252b76cf03a8d

Reply:
Fixed https://github.com/Qovery/console/pull/1671/commits/aab1ada750b7570ce08e86c1909a39f59d7e44e0